### ISSUE TITLE
Concurrent connections

### ISSUE THREAD

**Issue by bromeroarbelaez** [pierre-emmanuelJ/iptv-proxy#110]
Hello friend, I have seen that when you establish 2 or more concurrent connections to the proxy, the connections interfere. Reviewing the log, you can see that for each connection it pulls the other connections and what you see in the meantime is the client's buffer, that is, it does not support several simultaneous connections.

Regards

**Comment by Iucasmaia** [comment]
yes, if I open a channel on each device and change the channel on one of them, the device that changed the channel has preference and doesn't keep crashing... the other one keeps crashing a lot!

**Comment by XBS-Nathan** [comment]
we get any fixes for this afterwards?
